active/InformationBar/InformationBar.md

```diff
@@ -0,0 +1,512 @@
+
+## Is this the right control?
+Use an InfoBar control when a user should be informed of, acknowledge, or take action on a message. By default the notification will remain in the content area until dismissed by the user but will not necessarily break user flow.
+
+An InfoBar will take up space in your layout and behave like any other child elements. It will not cover up other content or float on top of it.
+
+Do not use an InfoBar control to confirm or respond directly to a user action, for time-sensitive alerts, or for non-essential messages.
+
+### Remarks
+Use an InfoBar that dismiss via the user or when the status is resolved for scenarios that **directly** impact app perception or experience ⚠
+
+- Internet connectivity lost
+- Error while saving a document when triggered automatically, not related to specific user action
+- No microphone plugged in when attempting to record
+- Can't connect to your phone
+- The subscription to the application is expired
+
+Use an InfoBar that dismiss via the user or a timer for scenarios that **indirectly** impact app perception or experience ℹ
+
+- A call has begun recording
+- Update applied with link to 'Release Notes'
+- The terms of service have been updated and require acknowledgement
+- An app-wide backup has successfully, asynchronously completed
+- The subscription to the application is close to expiring
+
+
+#### When should a different control be used?
+
+There are some scenarios where a ContentDialog, Flyout, or TeachingTip may be more appropriate to use.
+
+- For scenarios where a persistent notification is not needed, e.g. displaying information in context of a specific UI element, a [Flyout](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/dialogs-and-flyouts/flyouts) is a better option. 
+- For scenarios where the application is confirming a user action, showing information the user ***must*** read, use a [ContentDialog](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/dialogs-and-flyouts/dialogs).
+  - Additionally, if a status change to the app is so severe that it needs to block all further ability for the user to interact with the app, use a ContentDialog.
+- For scenarios where a notification is a transient teaching moment, a [TeachingTip](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/dialogs-and-flyouts/teaching-tip) is a better option.
+
+For more info about choosing the right notification control, see the [Dialogs and Flyouts](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/dialogs-and-flyouts/) article.
+
+# Examples
+
+An info bar can have several configurations, here are some notable ones.
+
+When an InfoBar is conveying information of a common severity, like displaying an error or warning, it can be set to have one of many severity levels to use consistent [Fluent](https://docs.microsoft.com/en-us/windows/apps/fluent-design-system) styling for it's identifiers.
```

Comment:
I don't understand the connection between the text and the picture. Is this example using a severity level? I would just remove this first section, up to the "Create an InfoBar" example, as it's redundant of the later examples. It provides an introduction/overview of the follow examples, but that's not a style that we use in the docs. It's better to jump right in to the first example.